data/sql/updates/pending_db_world/rev_1591204377400264600.sql

```diff
@@ -0,0 +1,40 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1591204377400264600');
+
+-- Quest 3181 must not be taken for this gossip to take place
+UPDATE `creature_template` SET `gossip_menu_id`=50000 , `ScriptName`='' WHERE  `entry`=3836;
+
+DELETE FROM `gossip_menu` WHERE `MenuID` IN (50000, 50001, 50002, 50003, 50004, 50005, 50006, 50007);
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`) VALUES 
+(50007, 1839),
+(50006, 1838),
+(50005, 1837),
+(50004, 1836),
+(50003, 1835),
+(50002, 1834),
+(50001, 1833),
+(50000, 1833);
+
+DELETE FROM `gossip_menu_option` WHERE `MenuID` IN (50000, 50001, 50002, 50003, 50004, 50005, 50006);
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES 
+(50003, 0, 0, "Doesn't matter, i'm invulnerable.", 0, 1, 1, 50004, 0, 0, 0, NULL, 0, 0),
```

Comment:
```suggestion
(50003, 0, 0, "Doesn't matter, i'm invulnerable!", 0, 1, 1, 50004, 0, 0, 0, NULL, 0, 0),
```

Reply:
the `i` should be in `uppercase`

Reply:
Not according to gamepedia... I was wondering about it too.

Maybe we can find a video